apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/name-step/index.tsx

```diff
@@ -42,7 +41,6 @@ const NameStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onN
 						id="nux-launch-step__input"
 						className="nux-launch-step__input"
 						onChange={ updateTitle }
-						onBlur={ saveTitle }
 						value={ title }
```

Comment:
With the changes in the `useTitle()` hook, I can now see a TypeScript error related to the fact that the `title` variable can now be `undefined`

In Focused Launch, this issue was solved by passing `title || ''` instead (see https://github.com/Automattic/wp-calypso/blob/cba6c935b5ad4736a52c74712996ddba7a66cb82/packages/launch/src/focused-launch/summary/index.tsx#L537)

Reply:
Good spot. Fixed.